code/modules/plumbing/ducts.dm

```diff
@@ -4,13 +4,16 @@ All the important duct code:
 /code/datums/ductnet.dm
 */
 /obj/machinery/duct
-	name = "fluid duct"
+	name = "Fluid Duct"
 	icon = 'icons/obj/plumbing/fluid_ducts.dmi'
 	icon_state = "nduct"
 	layer = PLUMBING_PIPE_VISIBILE_LAYER
 
 	use_power = NO_POWER_USE
 
+	//category for plumbing service rcd
```

Comment:
```suggestion
	///category for plumbing service rcd
```

Reply:
I have a question.  What should be given document level comments and what should be given normal comments? What's the standard?

Reply:
`///` is for doc-comments, which are important to consumers

`//` is for normal comments, which are important to readers, but not consumers (like if it's just implementation details, or if it's not a proc/type level var declaration)

Reply:
Removed the variable completely as it can be inferred directly in the for loop